app/views/admin/collections/show.html.erb

```diff
@@ -25,6 +25,14 @@ Unless required by applicable law or agreed to in writing, software distributed
     <%= button_tag('Edit Collection Info', class: 'btn btn-default', data: {toggle:"modal", target:"#new_collection"}) if can? :update, @collection %>
     <br/><br>
     Or <a href='https://wiki.dlib.indiana.edu/display/VarVideo/Avalon+Collections+Guide#AvalonCollectionsGuide-UploadingviaBatch'><b>learn how to</b></a> create items using batch upload.
+    <br /><br />
```

Comment:
I'll put everything in <p> ... </p>, the spacing looks sufficient and it's semantically better.

Interesting to know about <br> vs <br /> (I learned the latter in an XML course I took around 10 years ago).